scripts/lldbinit.py

```diff
@@ -11,18 +11,133 @@
 # This script is only meant to be used from LLDB.
 import lldb  # type: ignore
 import os
+import re
+from typing import Any
 
 project_root = os.path.dirname(os.path.realpath(__file__))
 
 ci = lldb.debugger.GetCommandInterpreter()
 result = lldb.SBCommandReturnObject()
 
 
-def RunCommand(cmd: str) -> None:
+def RunCommand(cmd: str, print_command: bool = True) -> Any:
     """Runs a command and prints it to the console to show that it ran."""
-    print("(lldb) %s" % cmd)
+    if print_command:
+        print(f"(lldb) {cmd}")
     ci.HandleCommand(cmd, result)
+    return result.GetOutput()
 
 
 RunCommand(f"settings append target.source-map . {project_root}")
 RunCommand(f"settings append target.source-map /proc/self/cwd {project_root}")
+
+# Matches the output of `print Dump(...)` and captures the stuff from inside the
+# std::string while discarding the std::string type.
+dump_re = re.compile('\\(std::string\\) "((:?.|\n)+)"', re.MULTILINE)
```

Comment:
FWIW, a security scan flagged that this RE could have exponential backtracking -- clearly not a security issue for the LLDB script, and no idea if it's realistic or useful to fix, but wanted to pass it along.

Specifically the `"((:?.|\n)+)"`:

> This part of the regular expression may cause exponential backtracking on strings starting with '(std::string) "' and containing many repetitions of ':a'.